docs/api/context-bridge.md

```diff
@@ -84,6 +90,47 @@ contextBridge.exposeInMainWorld(
 )
 ```
 
+An example of complex API exposed in an isolated world is shown below:
```

Comment:
I don't think we need a _complex_ example here, an incredibly simple example would suffice.  E.g. only expose a single method in the isolated world

Reply:
Added a simple example for `exposeInIsolatedWorld`